src/classes/TestUtils.cls

```diff
@@ -141,4 +178,138 @@ global class TestUtils {
+	* @return The escalation team pairing
+	*/
+	global static rh_escal__EscalationTeam__c getEscalationTeam(rh_escal__Escalation__c esc) {
+		return new rh_escal__EscalationTeam__c(
+			rh_escal__Escalation__c = esc.Id
+		);
+	}
+
+	/**
+	* Gets an escalation team
+	*
+	* @param esc The escalation, contact the type, isInternal if contact is internal
+	* @return rh_escal__EscalationTeam__c object
+	*/
+	global static rh_escal__EscalationTeam__c getEscalationTeam(rh_escal__Escalation__c esc, Contact contact, Boolean isInternal) {
+		rh_escal__EscalationTeam__c escTeam = getEscalationTeam(esc);
+		escTeam.rh_escal__Contact__c = contact.Id;
+		escTeam.rh_escal__Internal__c = isInternal;
+		return escTeam;
+	}
+
+	/**
+	* Gets an escalation team
+	*
+	* @param esc The escalation, user the type
+	* @return rh_escal__EscalationTeam__c object
+	*/
+	global static rh_escal__EscalationTeam__c getEscalationTeam(rh_escal__Escalation__c esc, User user) {
+		rh_escal__EscalationTeam__c escTeam = getEscalationTeam(esc);
+		escTeam.rh_escal__User__c = user.Id;
+		escTeam.rh_escal__Internal__c = true;
+		return escTeam;
+	}
+
+	/**
+	* Gets an escalation team
+	*
+	* @param esc The escalation, emailAdress the type
+	* @return rh_escal__EscalationTeam__c object
+	*/
+	global static rh_escal__EscalationTeam__c getEscalationTeam(rh_escal__Escalation__c esc, String emailAddress) {
+		rh_escal__EscalationTeam__c escTeam = getEscalationTeam(esc);
+		escTeam.rh_escal__Email__c = emailAddress;
+		return escTeam;
+	}
+
+	/**
+	* Queries an escalation team
+	*
+	* @param escalTeam The escalation team to query
+	* @return The queried escalation team
+	*/
+	global static rh_escal__EscalationTeam__c fetchEscalationTeam(rh_escal__EscalationTeam__c escalTeam) {
+		return [
+			select Name,
+				rh_escal__Contact__c,
+				rh_escal__Email__c,
+				rh_escal__Escalation__c,
+				rh_escal__Internal__c,
+				rh_escal__Type__c,
+				rh_escal__User__c
+			from rh_escal__EscalationTeam__c
+			where Id =:escalTeam.id
+		];
+	}
+
+	/**
+	* Gets a contact for an account with a given lastName
+	*
+	* @param a An account
+	* @param lName The last name
+	* @return A contact
+	*/
+	global static Contact getContact(Account a, String lName) {
+		String email = lName.replaceAll('[^a-zA-Z0-9._%+-]', '_');
+		email += '@test.org';
+
+		return new Contact(
+			AccountId = a.Id,
+			LastName = lName,
+			Email = email
+		);
+	}
+
+	/**
+	* Gets an admin user
+	*
+	* @return An admin user
+	*/
+	global static User getAdminUser() {
+		Profile adminProfile = PROFILE_NAME_MAP.get('System Administrator');
+
+		User admin = [
+			select Name,
+				ProfileId,
+				Profile.Name
+			from User
+			where ProfileId = :adminProfile.Id and
+				isActive = true and
+				LocaleSidKey = 'en_US'
+			limit 1
+		];
+
+		return admin;
+	}
+
+	/**
+	* Fetches a User with an Id
+	*
+	* @param u A userId
+	* @return A User
+	*/
+	global static User fetchUser(Id userId) {
+		return [
+			select
+				Alias,
```

Comment:
I am not using this method, I just added this for future use. I am removing this complete method.